src/components/kyb/ComplianceStep.tsx

```
@@ -0,0 +1,292 @@
+  // Expose validation function to parent component
+  useEffect(() => {
+    const allRequiredChecked = data.accurateInformation && 
+                              data.notIllegalActivities && 
+                              data.additionalInfoConsent && 
+                              data.privacyPolicyConsent;
+    
+    // Store validation status in parent component if needed
+    updateData({ 
+      ...data, 
+      _complianceStepValid: allRequiredChecked 
+    } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)